en/docs/extended_topics/consistent_mirrored.md

```diff
@@ -175,7 +175,7 @@ def train_job(
 ) -> tp.ListNumpy:
 ```
 
-* The data after dividing need to be stored in the `list` and pass to training functions. The number of elements in `list` need to be same as the **GPU number in training**. OneFlow will pass the data according to the order of the elements in `list ` to each GPU(the number i element in `list` is corresponding to number i GPU):
+* The data after dividing need to be stored in the `list` and pass to training functions. The number of elements in `list` need to be same as the **amount of devices in training**. OneFlow will pass the data according to the order of the elements in `list ` to each device(the number i element in `list` is corresponding to number i device):
```

Comment:
in the `list` and pass to training functions
->
in the `list` and passed to training functions